cpu/efm32/periph/timer.c

```diff
@@ -52,18 +65,42 @@ static inline bool _is_wtimer(timer_t dev)
 #endif
 }
 
-int timer_init(tim_t dev, unsigned long freq, timer_cb_t callback, void *arg)
+/**
+ * @brief   Check whether dev is using a LETIMER device
+ */
+static inline bool _is_letimer(timer_t dev)
 {
-    TIMER_TypeDef *pre, *tim;
+#if defined(LETIMER_COUNT) && LETIMER_COUNT > 0
+    return ((uint32_t) timer_config[dev].timer.dev) == LETIMER0_BASE;
+#else
+    (void) dev;
+    return false;
+#endif
+}
 
-    /* test if given timer device is valid */
-    if (dev >= TIMER_NUMOF) {
-        return -1;
-    }
+static void _letimer_init(tim_t dev, unsigned long freq)
+{
+    assert (freq == 32768);
```

Comment:
`assert(..)` needs an import of `#include "assert.h"`.

```suggestion
    assert(freq == 32768);
```